src/UI/Tester/WB.UI.Tester/Setup.cs

```diff
@@ -57,18 +57,6 @@ protected override void InitializeApp(IMvxApplication app)
                 CrashReporting.Init(appcenterKey);
             }
             
-            string arcgisruntimeKey = ApplicationContext.Resources.GetString(Resource.String.arcgisruntime_key);

```

Comment:
will World base map works after remove license key from tester?

Reply:
Offline maps should continue to work.